### PR TITLE
Contact form

### PR DESCRIPTION
**Pull request by medijdir**
## Highlights 🌟
in this project I have used: 
✔️ Form propriety 
✔️ HTML 
✔️  Use CSS flex to place elements on the page.
✔️  GitHub flow

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -175,6 +175,21 @@ <h5>Skills</h5>
         </ul>
         </section>
 
+        <section class="contact_form">
```
Comment (muneebulrehman) [inline index.html]: As per the Figma template provided to you, there is a certain gap between the `Skills` section and `Contact` section. However, in your design, there is no such gap. Kindly add some margin between the two. 

| Your design  | Figma design |
| ------------- | ------------- |
| image | image |

**Review by muneebulrehman — CHANGES_REQUESTED** [review]
Hi @medijdir , 

Good job so far! I see you put a lot of effort into this project. However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- No linter errors
- Descriptive PR
- Great job adding the form to your application.

## Required Changes ♻️
_Check the comments under the review._

- [ ] I see your partner made a commit. However, it is not a meaningful contribution to the project. Please ask your partner to contribute to the code base. You can let him fix the design for instance.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 2]
```diff
@@ -175,6 +175,19 @@ <h5>Skills</h5>
         </ul>
         </section>
 
-   
+        <section class="contact_form">
```
Comment (souad988) [inline index.html]: Kindly as requested by the previous reviewer fix the spacing issue between the about and contact sections , you can solve this problem in many ways and I suggest that you avoid setting fixed height for your components the good practice is to use padding and margin properties and let your content size reserve the height from dom flow , add padding-Bottom to the about section and for the color behind the contact form you can add subContainer to this section with properties of contact section and set contact section color to be white , feel free to use other solutions but  please make sure your project looks like the template before re-requesting review .

image (22)

**Review by souad988 — CHANGES_REQUESTED** [review]
Hi @medijdir  

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
Great efforts
Use of semantic html tags
Correct use of form action property 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 3]
```diff
@@ -250,8 +250,8 @@ ul {
 .About {
   background-color: #fff;
   border-radius: 0 50px 0 0;
-  height: 1000px;
-  width: 375px;
+  height: 72rem;
```
Comment (muneebulrehman) [inline style.css:253]: - Optional
Setting a fixed height in such a scenario is never a good idea. You can consider using `margin-bottom / padding-bottom` on this section or `margin-top` on the Contact section to put some distance between the two.

**Review by muneebulrehman — APPROVED** [review]
Hi @medijdir , 

Your project is complete!  There is nothing else to say other than... it's time to merge it 🥳
Congratulations! 🎉 

# Status: Approved✔️

## Highlights
- No linter errors
- Descriptive PR
- Good follow-up with the previous reviewer
- Good job with the design.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**